01-python/outdoors/lake.py

```
@@ -1,3 +1,36 @@
-def draw_lake():
-    print("lake not found")
-    return
\ No newline at end of file
+from random import random
+
+def draw_lake(numFish=3):
+  # Make sure user didn't put in a number that was too high
+  numFish = 8 if numFish > 8 else numFish
+  # Same for too low
+  numFish = 0 if numFish < 0 else numFish
```

same here - this is very dangerous

---

please raise an error instead

---

Makes sense. I must've been in a scripting mindset rather than a programming mindset 😁